.github/workflows/build.yml

```diff
@@ -60,6 +60,8 @@ jobs:
             | tr -d '[[:space:]]')
           export PLUGIN_FOLDER_FILENAME="elementor-${PACKAGE_VERSION}-${PLUGIN_VERSION}"
           export PLUGIN_ZIP_FILENAME="${PLUGIN_FOLDER_FILENAME}.zip"
+          composer install --optimize-autoloader
```

Comment:
```suggestion
          composer install --optimize-autoloader --prefer-dist --no-dev
```

Reply:
@Nevoss If we run with the flag --no-dev we won't have access to the PHP-scoper package in the dev dependencies to run the scoper command in the composer script 			"@php ./vendor/bin/php-scoper add-prefix --output-dir=./vendor_prefixed --config=php-scoper/dependency-injection.inc.php --force"